fdbserver/include/fdbserver/workloads/MockTLog.actor.h

```diff
@@ -0,0 +1,125 @@
+/*
+ * MockTLog.actor.h
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2021 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#if defined(NO_INTELLISENSE) && !defined(FDBSERVER_TEST_MockTLog_ACTOR_G_H)
+#define FDBSERVER_TEST_MockTLog_ACTOR_G_H
+#include "fdbserver/workloads/MockTLog.actor.g.h"
+#elif !defined(FDBSERVER_TEST_MockTLog_ACTOR_H)
+#define FDBSERVER_TEST_MockTLog_ACTOR_H
+
+#include <memory>
+#include <unordered_map>
+
+#include "fdbclient/FDBTypes.h"
+#include "fdbserver/LogSystem.h"
+#include "fdbserver/ResolverInterface.h"
+#include "fdbserver/TLogInterface.h"
+#include "fdbclient/StorageServerInterface.h"
+#include "flow/flow.h"
+
+#include "flow/actorcompiler.h" // has to be last include
+
+#pragma once
+
+struct TestTLogDriverOptions {
+	std::string diskQueueBasename;
+	std::string diskQueueExtension;
+	std::string kvStoreFilename;
+	std::string dataFolder;
+	std::string kvStoreExtension;
+	int64_t kvMemoryLimit;
+	int numTagsPerServer;
+	int numLogServers;
+	int numCommits;
+
+	explicit TestTLogDriverOptions(const UnitTestParameters& params) {
+		diskQueueBasename = params.get("diskQueueBasename").orDefault("folder");
+		diskQueueExtension = params.get("diskQueueFileExtension").orDefault("ext");
+		kvStoreExtension = params.get("diskQueueFileExtension").orDefault("fdr");
```

Comment:
Parameter "kvStoreExtension"?

Reply:
kvstore of type "RADIXTREE" has an "fdr" file extension.